homeassistant/components/viaggiatreno/sensor.py

```diff
@@ -174,7 +174,7 @@ async def async_update(self) -> None:
                 self._state = NO_INFORMATION_STRING
                 self._unit = ""
             else:
-                self._state = "Error: {}".format(res["error"])
+                self._state = "Error: " + res["error"]
```

Comment:
Same as before, if `res["error"]` isn't a string, this will fail.

Reply:
Done